server/conn.go

```diff
@@ -1507,17 +1507,21 @@ func (cc *clientConn) flush(ctx context.Context) error {
 
 func (cc *clientConn) writeOK(ctx context.Context) error {
 	msg := cc.ctx.LastMessage()
-	return cc.writeOkWith(ctx, msg, cc.ctx.AffectedRows(), cc.ctx.LastInsertID(), cc.ctx.Status(), cc.ctx.WarningCount())
+	return cc.writeOkWith(ctx, msg, cc.ctx.AffectedRows(), cc.ctx.LastInsertID(), cc.ctx.Status(), cc.ctx.WarningCount(), false)
 }
 
-func (cc *clientConn) writeOkWith(ctx context.Context, msg string, affectedRows, lastInsertID uint64, status, warnCnt uint16) error {
+func (cc *clientConn) writeOkWith(ctx context.Context, msg string, affectedRows, lastInsertID uint64, status, warnCnt uint16, eof bool) error {
```

Comment:
IMHO it's a better idea to change the signature to(more straightforward):

```
func (cc *clientConn) writeOkWith(ctx context.Context, header byte, msg string, affectedRows, lastInsertID uint64, status, warnCnt uint16)
```
This is what I did in #32119, what do you think :)

Reply:
Done in 78772e40fb9ffb5cab0e939a95bc062d9ccbba29